Riot/Managers/LocalAuthentication/LocalAuthenticationService.swift

```diff
@@ -57,12 +57,29 @@ class LocalAuthenticationService: NSObject {
         return (systemUptime - appLastActiveTime) > pinCodePreferences.graceTimeInSeconds
     }
     
+    var shouldShowInactiveScreen: Bool {
```

Comment:
To avoid to make `LocalAuthenticationService` depends on `BiometricsAuthenticationPresenter ` which is a view  management class, it can be interesting to define this boolean value outside to make those class independent. At the moment in `LegacyAppDelegate` we can have local variable: `BOOL shouldShowInactiveScreen = self.localAuthenticationService.isProtectionSet && BiometricsAuthenticationPresenter.isPresenting` for example

Reply:
Updated